test_console_utility/test_process_file_path.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import argparse
+import os
+import tempfile as tf
+from unittest.mock import Mock
+from processor import ArgumentProcessor
+
+"""Test handling absolute, relative and invalid file paths."""
+
+
+TEST_DIR_1 = tf.mkdtemp()
+TEST_DIR_2 = "test_output"
+
+test_params_valid = [
+    (TEST_DIR_1, TEST_DIR_1),
+    (TEST_DIR_2, "/Users/jprzygoda/data_generator_app/test_console_utility/test_output"),
```

Comment:
This path works only in your environment, please replace this path with the parametrized one.